src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <Age age={person.age} />
+    <Partner
+      isMarried={person.isMarried}
+      sex={person.sex}
+      partnerName={person.partnerName}
+    />
+  </section>
+);
```

Comment:
looks good actually, let's see the alternative formatting:

```suggestion
export const Person = ({ person }) => {
  const { age, sex, partnerName, isMarried } = person;

    return (
      <section className="Person">
        <h2 className="Person__name">
          {`My name is ${name}`}
        </h2>
        
        <Age age={age} />
        
        <Partner
          isMarried={isMarried}
          sex={sex}
          partnerName={partnerName}
        />
      </section>
    );
};
```

It depends on the formatting conventions of the company, but I prefer it this way.  What changed:
- added `person` object destructuring in order to shorten the syntax when passing to props
- added one line in between components of the same nesting level
- moved the text from `h2` to a separate line in order to improve readability

For now we can treat it as a matter of taste, you may choose what looks better for you